examples/gno.land/r/demo/foo20-airdrop/airdrop.gno

```diff
@@ -0,0 +1,39 @@
+package foo20airdrop
+
+import (
+	"gno.land/p/demo/airdrop"
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/merkle"
+	"gno.land/r/demo/foo20"
+)
+
+var (
+	token grc20.IGRC20 = foo20.GRC20()
+
+	// admin std.Address = "g1sw5xklxjjuv0yvuxy5f5s3l3mnj0nqq626a9wr" // albttx.gno
+
+	foo20airdrop *airdrop.MerkleAirdrop
+)
+
+func RegisterMerkleRoot(root string) {
+	if foo20airdrop != nil {
+		panic("foo20 airdrop merkle root is already registered")
+	}
+	foo20airdrop = airdrop.NewMerkleAirdrop(root, token)
+}
+
+func Claim(data airdrop.AirdropData, proofs []merkle.Node) {
+	err := foo20airdrop.Claim(data, proofs)
+	if err != nil {
+		panic(err.Error())
+	}
+}
+
+func TotalClaimed() uint64 {
+	return foo20airdrop.TotalClaimed()
+}
+
+// for tests purpose
+func reset() {
```

Comment:
Can't you modify `foo20airdrop` from the test directly, since they are in the same package? You can then drop this method entirely